crates/zkwasm/src/circuits/utils/image_table.rs

```diff
@@ -83,8 +82,8 @@ pub fn image_table_offset_to_memory_location(offset: usize) -> (LocationType, u3
  */
 #[derive(Debug)]
 pub struct ImageTableLayouter<T> {
-    pub(crate) initialization_state: InitializationState<T, T>,
-    pub(crate) static_frame_entries: [(T, T); STATIC_FRAME_ENTRY_NUMBER],
```

Comment:
Comment: Remove `enable` cell in (pre and post) image table since `encode` must be 0 if `enable` is 0 in frame table.